.github/pull_request_template.md

```diff
@@ -4,38 +4,57 @@
   If a section of the PR template does not apply to this PR, then delete that section.
  -->
 
-## Motivation
+## What type of PR is this?
 
-_(fill-in or delete this section)_
+_(REQUIRED)_
+
+<!--
+  /kind bug
+  /kind cleanup  
+  /kind documentation
+  /kind feature
+ -->
+
+## What this PR does / why we need it:
+
+_(REQUIRED)_
 
 <!--
   What goal is this change working towards?
+  Provide a bullet pointed summary of how each file was changed.
+  Briefly explain any decisions you made with respect to the changes.
+  Include anything here that you didn't include in *Release Notes*
+  above, such as changes to CI or changes to internal methods.
+-->
+
+## Which issue(s) this PR fixes:
+
+_(REQUIRED)_
+<!--
   If this PR fixes one of more issues, list them here.
   One line each, like so:
     Fixes #123
     Fixes #39
 -->
 
-## Release Notes
+## Special notes for your reviewer:
 
 _(fill-in or delete this section)_
 
 <!--
-  If this PR makes user facing changes, please describe them here.
-  This description will be copied into the release notes, whenever
-  the next version is released. Keep this section short, and
-  focus on high level changes.
+   Is there any particular feedback you would / wouldn't like?
+   Which parts of the code should reviewers focus on?
 -->
 
-## Changes
+## Release Notes
 
-_(fill-in or delete this section)_
+_(REQUIRED)_
```

Comment:
We would have to use the release notes block like in the k8s template to let the parser work:

````
```release-note

```
````
We can also indicate that `NONE` in the note is the correct way to omit release notes.

Reply:
Fixed.